mobile-app/app/components/CollateralFactorTag.tsx

```diff
@@ -0,0 +1,26 @@
+import { tailwind } from "@tailwind";
+import BigNumber from "bignumber.js";
+import { ThemedTextV2, ThemedViewV2 } from "./themed";
+
+export function CollateralFactorTag({
+  factor,
+}: {
+  factor?: string;
+}): JSX.Element | null {
+  const DEFAULT_FACTOR = new BigNumber(1);
+  return factor !== undefined && !DEFAULT_FACTOR.isEqualTo(factor) ? (
```

Comment:
can we simplify to this?
```suggestion
  return !DEFAULT_FACTOR.isEqualTo(factor) ? (
```

Reply:
What if factor is undefined? Can I do this ?
```return !DEFAULT_FACTOR.isEqualTo(factor ?? 0) ? (```

Reply:
Eh no, i dont think so, then it will be possible to have `undefinedx`